services/core-api/app/api/users/minespace/resources/minespace_user.py

```diff
@@ -33,8 +33,8 @@ def get(self):
         if not is_admin and mine_guid is None:
             raise BadRequest("mine_guid is a required argument")
         
-        if request.args.get('email_or_username'):
-            ms_users = [MinespaceUser.find_by_email(request.args.get('email_or_username'))]
+        if request.args.get('bceid_username'):
+            ms_users = [MinespaceUser.find_by_username(request.args.get('bceid_username'))]
```

Comment:
Hmm something feels a bit odd with this logic here reading this whole function. I think we should probably also filter this by the mine_guid you pass in if you're a minespace proponent?

Reply:
this is some 5 year old code, which, based on the FE code, I think is actually just _dead_. There's no calls to the API matching this resource where we do a GET with anything other than mine_guid. I'll take it out.